.travis.yml

```diff
@@ -30,3 +29,7 @@ script:
 
 after_success:
   - test/update-coveralls
+
+cache:
```

Comment:
I didn't try too hard with this, I am sure we can do even more. I just picked the lowest hanging fruit here.